include/clang/Driver/Options.td

```diff
@@ -2024,6 +2024,9 @@ def mglibc : Flag<["-"], "mglibc">, Group<m_libc_Group>, Flags<[HelpHidden]>;
 def muclibc : Flag<["-"], "muclibc">, Group<m_libc_Group>, Flags<[HelpHidden]>;
 def module_file_info : Flag<["-"], "module-file-info">, Flags<[DriverOption,CC1Option]>, Group<Action_Group>,
   HelpText<"Provide information about a particular module file">;
+def msandbox : Flag<["-"], "msandbox">, Group<m_Group>,
+  HelpText<"CHERI Pure Capability ABI generated code will be run in a sandbox (CHERI only)">,
```

Comment:
`-msandbox` shouldn't imply `-mabi=purecap` - I think n64 code is currently broken in sandboxes, but we should support hybrid ABI code in sandboxes again.